frontend/src/components/InstanceForm.vue

```diff
@@ -864,8 +845,9 @@ const doUpdate = () => {
               type: dataSource.type,
               username: dataSource.username,
               password: dataSource.password,
-              hostOverride: dataSource.hostOverride,
-              portOverride: dataSource.portOverride,
+              host: dataSource.host,
```

Comment:
Seems we shouldn't check the `dataSource.type` anymore, we should also create the "ADMIN" data source. Similar for line 869
The frontend logic here needs a refactor.

Reply:
Oh I found we only update the`InstancePatch`...Do we also need to update the `InstanceCreate`?

Reply:
The current `instance` table still has these fields. So let's update `InstanceCreate` in the follow-up PRs.
https://github.com/bytebase/bytebase/blob/main/store/migration/dev/LATEST.sql#L287